package.json

```diff
@@ -15,11 +15,12 @@
 		"bump": "node ./scripts/bump.js",
 		"lint": "eslint \"src/**/*.js?(x)\" \"tests/**/*.jsx\"",
 		"pretest": "npm run lint",
-		"test": "npm run test:browser && npm run test:ssr",
+		"test": "npm run test:dry",
+		"test:all": "node ./scripts/react-tester.js --react all",
 		"test:browser": "karma start",
```

Comment:
I wonder if `test:browser` still makes sense, since from this PR we use react-tester as an entry point for all test runs :thinking:

Reply:
It's used from within `react-tester.js` as `npm` command.

Reply:
Ahhh, somehow missed it :+1: